packages/wonder-blocks-form/src/components/__docs__/labeled-text-field.stories.js

```diff
@@ -528,6 +528,27 @@ CustomStyle.parameters = {
     },
 };
 
+export const WithMarkup: StoryComponentType = (args) => {
+    return (
+        <LabeledTextField
+            {...args}
+            label="Name"
+            description={
+                <span>
+                    Please enter your <strong>name</strong>
+                </span>
```

Comment:
**suggestion:** It would be better including an example using WB components.

e.g.
```
<LabelSmall>Please enter your <strong>name</strong><LabelSmall>
```

or

```
<View>something custom...</View>
```